tests/fixtures.py

```diff
@@ -725,7 +725,7 @@ def parametrize(params, values, **kwargs):
         ("b9f5b", "file", OS(OS_ANY), True),
         ("b9f5b", "file", Format(FORMAT_DOTNET), True),
         ("hello-world", "file", capa.features.file.FunctionName("HelloWorld::Main"), True),
-        ("hello-world", "file", capa.features.file.FunctionName("HelloWorld::.ctor"), True),
+        ("hello-world", "file", capa.features.file.FunctionName("HelloWorld::ctor"), True),
         ("hello-world", "file", capa.features.file.FunctionName("HelloWorld::.cctor"), False),
```

Comment:
is there a reason we don't want the dot for `ctor` but for `cctor`?

Reply:
good catch; we should remove `.` from both. I'll make this change.